keyboards/herevoland/buff75/matrix_diagram.md

```diff
@@ -0,0 +1,21 @@
+# Matrix Diagram for Buff75
```

Comment:
> This PCB has two versions: hotswap and solder.

If only some of the possible layout can apply to either, then the expectation is that the keyboard has 2 sub-revisions, 
```
keyboards/herevoland/buff75/soldered/keyboard.json
keyboards/herevoland/buff75/hotswap/keyboard.json
```

Reply:
Just deleting the `matrix_diagram.md` file does not exclude you from making these changes.

Your product photo clearly shows multiple available layouts.